khal/cli.py

```diff
@@ -480,9 +486,12 @@ def isatty(_file):
 
     @cli.command()
     @multi_calendar_option
+    @no_mouse_option
     @click.pass_context
-    def interactive(ctx, include_calendar, exclude_calendar):
+    def interactive(ctx, include_calendar, exclude_calendar, no_mouse):
         '''Interactive UI. Also launchable via `ikhal`.'''
+        if no_mouse:
+            ctx.obj['conf']['default']['enable_mouse'] = False
```

Comment:
I'm overriding the config value when the `--no-mouse` flag is used. Not sure this is the best way, but seems to work.